compiler/quilt/test/test_store.py

```diff
@@ -0,0 +1,60 @@
+"""
```

Comment:
This test looks good, but I think it makes more sense to move the parse_package helpers out of store.py (they're called in multiple places and aren't related to the PackageStore) than to create this new test file.

Reply:
That works for me.   Where should I move it, though?  There's no 'test_utils.py', and it seems tools.utils would be a good spot for it.

Reply:
On second thought, just leave it as is for now and we can move both the helpers and tests in a separate PR.

Reply:
That sounds good to me.  I was noticing the same thing, btw.